scripts/templates/exp_feat.rst.mako

```diff
@@ -0,0 +1,82 @@
+<%text><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+</%text>
+.. _experimental-${name}:
+
+================================================================================
+${" ".join(name.split("-")).title()}
+================================================================================
+
+.. _warning:
+
+    Experimental features:
+
+    *   May be replaced, updated, or removed at any time.
+    *   Do not require maintaining API/ABI stability of their own additions over
+        time.
+    *   Do not require conformance testing of their own additions.
+
+
+--------------------------------------------------------------------------------
```

Comment:
Other than the main page title I don't think these lines above the titles are necessary